microcosm_flask/conventions/relation.py

```diff
@@ -99,6 +98,12 @@ def configure_relation(graph, from_obj, to_obj, mappings, path_prefix=""):
     Register relation endpoint(s) between two resources.
 
     """
+    ns = Namespace(
```

Comment:
Unlike the CRUD configuration function, there's no good way to have the caller pass a `Namespace` without changing the call signature in a backwards incompatible way.

My guess is that we'll choose to break compatibly (it's early days for this library), but I didn't want to do so without discussion.